docs/docs/apis/actions.md

```diff
@@ -5,20 +5,22 @@ title: Actions
 This page describes the options you have when writing ZITADEL actions scripts.
 
 ## Language
+
 ZITADEL interpretes the scripts as JavaScript.
 Make sure your scripts are ECMAScript 5.1(+) compliant.
 Go to the [goja GitHub page](https://github.com/dop251/goja) for detailed reference about the underlying library features and limitations.
 
+Actions provide a defined set of libraries. The provided libraries vary depending on trigger types.
+
 Actions do not have access to any libraries yet.
 Also, sending HTTP requests is not supported yet.
 [We plan to add such features in the future](https://zitadel.com/roadmap).
```

Comment:
I thought we provide a method to do http requests?

Reply:
sorry I forgot to delete this file